patient_case_summary/streamlit.py

```
@@ -0,0 +1,123 @@
+    llm = llm or OpenAI(model="gpt-4-turbo")  # "gpt-4o")
+
+    workflow = GuidelineRecommendationWorkflow(
+        guideline_retriever=retriever,
+        llm=llm,
+        verbose=True,
+        timeout=None,  # don't worry about timeout to make sure it completes
+    )
+
+    handler = workflow.run(patient_json_path= patient_info_path)
+    async for event in handler.stream_events():
+        if isinstance(event, LogEvent):
```

Ruff (0.8.2)

`LogEvent` may be undefined, or defined from star imports

(F405)